frontend/src/components/samples/details/SampleDetailsContent.js

```diff
@@ -180,10 +182,11 @@ const SampleDetailsContent = ({
     })
   }, [sample])
 
-  const tabsStyle = {
-    marginTop: 8,
-    overflow: activeKey === 'lineage' ? 'hidden' : 'auto'
-  }
+  const tabsClass = classNames({
```

Comment:
This is generating bad class names: `ant-tabs ant-tabs-top ant-tabs-large ant-tabs-card .ant-tabs .ant-tabs-content.ant-tabs-content-top .lineage-tab-active`. Those dots shouldn't be there, and it's duplicating the properties that are already there. I'm not sure how classNames works, but it's not doing the right thing here.